apps/cmitls/Makefile

```diff
@@ -2,11 +2,12 @@ MITLS_HOME ?= ../..
 FSTAR_HOME ?= ../../../FStar
 MLCRYPTO_HOME ?= ../../../MLCrypto
 
-include $(FSTAR_HOME)/ulib/ml/Makefile.include
-
 UNAME=$(shell uname)
 MARCH?=x86_64
 
+nullstring := #empty variable
```

Comment:
Are these used anywhere?

Reply:
nullstring is used inside the "space" variable.  This is an unhappy-but-popular solution for making make concatenate a list together without spaces between the parts.  Needed to take a list of paths and make it into a colon-separated list, suitable for LD_LIBRARY_PATH.

Reply:
I'm confused, the string "space" does not appear elsewhere in apps/*/Makefile? Is this used somewhere downstream in `includ`ed Makefiles?

Reply:
It is consumed on the next line in that Makefile.:
+space := $(nullstring) $(nullstring) #one space

Reply:
Yes, but "space" is not consumed by anything, or is it?